yarn-project/end-to-end/src/e2e_key_rotation.test.ts

```diff
@@ -31,13 +31,12 @@ describe('e2e_key_rotation', () => {
   let aztecNode: AztecNode;
   let pxeA: PXE;
   let pxeB: PXE;
-  let walletA: Wallet;
-  let walletB: Wallet;
+  let walletA: AccountWallet;
+  let walletB: AccountWallet;
```

Comment:
Being forced to replace interface with an instance seems incorrect. Why not update the Wallet interface?